src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Windows/PkcsPalWindows.cs

```diff
@@ -214,18 +214,37 @@ public override T GetPrivateKeyForDecryption<T>(X509Certificate2 certificate, bo
 
                 if (keySpec == CryptKeySpec.CERT_NCRYPT_KEY_SPEC)
                 {
+
```

Comment:
why the extra blank line?

Reply:
😳 it looked nice there?

I suspect I wrote something there then refactored / removed the code but left a blank line by accident. Removed.